p2p/src/utils/mod.rs

```diff
@@ -33,11 +33,12 @@ impl<P: PublicKey> Manager for StaticManager<P> {
     type PublicKey = P;
     type Peers = Ordered<P>;
 
-    async fn update(&mut self, _: u64, peers: Ordered<P>) {
-        self.peers = peers;
+    async fn update(&mut self, _: u64, _: Ordered<P>) {
+        unreachable!("updates are not supported");
```

Comment:
nit: this is technically reachable if someone uses this incorrectly right? or is this type not public outside the crate? If reachable, then probably `panic!` is more appropriate